packages/console/src/components/Table/index.tsx

```diff
@@ -69,7 +75,14 @@ const Table = <
               <TableError columns={columns.length} content={errorMessage} onRetry={onRetry} />
             )}
             {!isLoading && !hasData && placeholder && (
-              <TableEmpty columns={columns.length}>{placeholder}</TableEmpty>
+              <TableEmpty
+                columns={columns.length}
+                title={placeholderTitle}
+                content={placeholderContent}
+                image={placeholderImage}
+              >
+                {placeholder}
+              </TableEmpty>
```

Comment:
what is the difference between placeholder content and placeHolder

Reply:
Refactored the old logic; the 'content' should be a 'description'.